PowerPointLabs/Test/UnitTest/SyncLab/SyncLabPositionTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Office.Interop.PowerPoint;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using PowerPointLabs.SyncLab.ObjectFormats;
+
+namespace Test.UnitTest.SyncLab
+{
+    [TestClass]
+    public class SyncLabPositionTest : BaseSyncLabTest
```

Comment:
@kai33 I have been trying to do the FTs, but I am unsure of how to detect the pop up window after clicking the "Copy" button on the Sync Pane.. any suggestions?